Packs/Netskope/Integrations/NetskopeEventCollector/NetskopeEventCollector.yml

```diff
@@ -0,0 +1,89 @@
+category: Analytics & SIEM
+commonfields:
+  id: NetskopeEventCollector
+  version: -1
+configuration:
+- display: Server URL
+  name: url
+  required: true
+  type: 0
+- hidden: false
+  name: credentials
+  required: true
+  type: 9
+- additionalinfo: The API version to use (v1 or v2).
+  defaultvalue: v2
+  display: API Version
+  hidden: false
+  name: api_version
+  required: false
+  type: 0
```

Comment:
So I don't have access to the UI. 
But in the step of getting the API key, if the user have a v1 it will say on the settings page -Rest API v1
same for v2.

Reply:
Let's try and look for docs on where to get that key.